### PR TITLE
Removed Obsolete layout params

### LINKED ISSUE #1400: Obsolete layout params
**Summary:** 

There are some layout_param which not defined for the given layout, meaning it has no effect. 
This usually happens when you change the parent layout or move view code around without updating the layout params. 

This will cause useless attribute processing at runtime and is misleading for others reading the layout so the parameter should be removed.

### PR DESCRIPTION
**Pull request by RajaVamsi11**
Fixes #1400 

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `MifosStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

File: mifosng-android/src/main/res/layout/row_client_list_item.xml [hunk 1]
```diff
@@ -28,7 +28,6 @@
             android:id="@+id/tv_product_short_name"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
-            android:layout_below="@+id/tv_clientName"
             android:text=""
```
Comment (dreadpool2) [inline mifosng-android/src/main/res/layout/row_client_list_item.xml]: Isn't this obsolete too?

Comment (RajaVamsi11) [reply mifosng-android/src/main/res/layout/row_client_list_item.xml]: Yeah, it is. Either we could delete it or give some text like other Text Views

Comment (RajaVamsi11) [reply mifosng-android/src/main/res/layout/row_client_list_item.xml]: I included the text for that text view.

**Comment by Aditya-gupta99** [comment]
Closing this PR since the XML file has been updated

**Pull request closed without merging**